imports/ui/components/CardList.jsx

```diff
@@ -1,22 +1,23 @@
 import React, { Component } from "react";
 import Card from "./Card";
 import Search from "./Search";
+import SearchBar from './SearchBar';
```

Comment:
Double quotation to match the rest of the file

Reply:
Fixed